### PR TITLE
Add blog post "Contributing Upstream"

### PR DESCRIPTION
**Pull request by osfrickler**

### CODE REVIEW

File: _i18n/en/_posts/blog/2024-09-09-upstream-contribution.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title: "Contributing Upstream - how and why"
+author:
+  - "Dr. Jens Harbott"
+avatar:
+  - "jhorstmann.jpg"
```
Comment (osfrickler) [inline _i18n/en/_posts/blog/2024-09-09-upstream-contribution.md:7]: was testing to fix the build error, will use the correct one for the next iteration

**Comment by fkr** [comment]
@osfrickler I'm already assigned for review but the state is still 'Draft'. Are you seeking feedback on the current state?

**Comment by osfrickler** [comment]
Yes, Christian asked me to do so, I was assuming that he had already talked to you about this, that's why I didn't add some explaining comment.

There are some sections yet to be filled in, but what's there could stay as it is if people agree, so feedback on that would be welcome. I'll also push another update with more content later today.

**Comment by fkr** [comment]
This reads really well. I do think that the part about Release Engineering and TC could use some more elaboration - other than that, really good post.
I cherry-picked the fixes to the Readme (https://github.com/SovereignCloudStack/website/pull/1016) since they do not belong into this PR.

**Comment by berendt** [comment]
@fkr This is ready and can be finalised.

**Comment by fkr** [comment]
Cool. Pushed to staging, when it is visible there, I'll do a last review.

**Review by fkr — APPROVED** [review]

**Pull request merged**